src/Inventory.zig

```diff
@@ -319,7 +321,12 @@ pub const Sync = struct { // MARK: Sync
 			defer mutex.unlock();
 			const inventory = ServerInventory.init(len, typ, source, .externallyManaged);
 			inventories.items[inventory.inv.id] = inventory;
-			inventory.inv.loadFromZon(zon);
+			switch(zon) {
+				.object => inventory.inv.loadFromZon(zon),
+				.string, .stringOwned => |str| inventory.inv.fromBase64(str),
```

Comment:
Conversion from base64 should be handled on the outside, in the player data parsing/storing code.
Otherwise we will end up having to add a copy of this function once chests are stored.